web/src/components/DandisetList.vue

```diff
@@ -144,3 +139,20 @@ export default defineComponent({
   },
 });
 </script>
+
+<style scoped>
+.wrap-text {
+  -webkit-line-clamp: unset !important;
+}
+.v-list-item__title {
+  white-space: normal;
+}
+
+.v-list-item {
+  border-bottom: 1px solid #eee;
+}
+
+.v-list--three-line .v-list-item .v-list-item__content, .v-list-item--three-line .v-list-item__content {
```

Comment:
appears to be a linting error on this line being too long.

Reply:
Oh thanks, I'm not sure why this didn't throw an error in my `yarn run serve` but it is resolved now